scripts/choose_staking.py

```diff
@@ -25,85 +25,52 @@
+}
+NO_STAKING_PROGRAM_ENV_VARIABLES = {
+    "USE_STAKING": "false",
+    "STAKING_PROGRAM": NO_STAKING_PROGRAM_ID,
+    "AGENT_ID": "25",
+    "CUSTOM_SERVICE_REGISTRY_ADDRESS": "0x9338b5153AE39BB89f50468E608eD9d764B755fD",
+    "CUSTOM_SERVICE_REGISTRY_TOKEN_UTILITY_ADDRESS": "0xa45E64d13A30a51b91ae0eb182e88a40e9b18eD8",
+    "MECH_CONTRACT_ADDRESS": "0x77af31De935740567Cf4fF1986D04B2c964A786a",
+    "CUSTOM_OLAS_ADDRESS": ZERO_ADDRESS,
+    "CUSTOM_STAKING_ADDRESS": ZERO_ADDRESS,
+    "MECH_ACTIVITY_CHECKER_CONTRACT": ZERO_ADDRESS,
+    "MIN_STAKING_BOND_OLAS": "0",
+    "MIN_STAKING_DEPOSIT_OLAS": "0",
+}
 
 # Information stored in the "deployment" key is used only to retrieve "stakingTokenInstanceAddress" (proxy)
 # and "stakingTokenAddress" (implementation). The rest of the parameters are read on-chain.
 STAKING_PROGRAMS = {
-    "no_staking": {
-        "name": "No staking",
-        "description": "Your Olas Predict agent will still actively participate in prediction markets, but it will not be staked within any staking program.",
-        "deployment": {
-            "stakingTokenAddress": "0x43fB32f25dce34EB76c78C7A42C8F40F84BCD237",
-            "stakingTokenInstanceAddress": "0x43fB32f25dce34EB76c78C7A42C8F40F84BCD237"
-        }
-    },
-    "quickstart_beta_hobbyist": {
-        "name": "Quickstart Beta - Hobbyist",
-        "description": "The Quickstart Beta - Hobbyist staking contract offers 100 slots for operators running Olas Predict agents with the quickstart. It is designed as a step up from Coastal Staker Expeditions, requiring 100 OLAS for staking. The rewards are also more attractive than with Coastal Staker Expeditions.",
-        # https://github.com/valory-xyz/autonolas-staking-programmes/blob/main/scripts/deployment/globals_gnosis_mainnet_qs_beta_hobbyist.json
-        "deployment": {
-            "stakingTokenAddress": "0xEa00be6690a871827fAfD705440D20dd75e67AB1",
-            "stakingTokenInstanceAddress": "0x389B46c259631Acd6a69Bde8B6cEe218230bAE8C"
-        }
-    },
-    "quickstart_beta_expert": {
-        "name": "Quickstart Beta - Expert",
-        "description": "The Quickstart Beta - Expert staking contract offers 20 slots for operators running Olas Predict agents with the quickstart. It is designed for professional agent operators, requiring 1000 OLAS for staking. The rewards are proportional to the Quickstart Beta - Hobbyist.",
-        # https://github.com/valory-xyz/autonolas-staking-programmes/blob/main/scripts/deployment/globals_gnosis_mainnet_qs_beta_expert.json
-        "deployment": {
-            "stakingTokenAddress": "0xEa00be6690a871827fAfD705440D20dd75e67AB1",
-            "stakingTokenInstanceAddress": "0x5344B7DD311e5d3DdDd46A4f71481bD7b05AAA3e"
-        }
-    },
+    NO_STAKING_PROGRAM_ID: ZERO_ADDRESS,
+    "quickstart_beta_hobbyist": "0x389B46c259631Acd6a69Bde8B6cEe218230bAE8C",
+    "quickstart_beta_expert": "0x5344B7DD311e5d3DdDd46A4f71481bD7b05AAA3e",
```

Comment:
Simplified this data structure - we only need the contract address. All remaining information can be fetch on-chain.